src/rapidata/rapidata_client/filter/user_score_filter.py

```diff
@@ -13,14 +13,15 @@ class UserScoreFilter(RapidataFilter):
     Args:
         lower_bound (float): The lower bound of the user score.
         upper_bound (float): The upper bound of the user score.
+        dimension (str): The dimension of the userScore to be considerd for the filter. Default is "global".
 
     Example: 
         ```python
         UserScoreFilter(0.5, 0.9)
         ```
         This will only show the order to users that have a UserScore of >=0.5 and <=0.9
     """
-    def __init__(self, lower_bound: float = 0.0, upper_bound: float = 1.0):
+    def __init__(self, lower_bound: float = 0.0, upper_bound: float = 1.0, dimension: str = "global"):
```

Comment:
can you by default not provide any dimension at all? this way we can control the default value simply from the backend and don't have to worry about the evaluators implementation.